typescript/ws-protocol/src/FoxgloveServer.ts

```diff
@@ -154,13 +193,19 @@ export default class FoxgloveServer {
       subscriptions: new Map(),
       subscriptionsByChannel: new Map(),
       advertisements: new Map(),
+      parameterSubscriptions: new Set<string>(),
     };
     this.clients.set(connection, client);
 
     this.send(connection, {
       op: "serverInfo",
       name: this.name,
-      capabilities: [ServerCapability.clientPublish, ServerCapability.time],
+      capabilities: [
+        ServerCapability.clientPublish,
+        ServerCapability.time,
+        ServerCapability.parameters,
+        ServerCapability.parametersSubscribe,
+      ],
```

Comment:
Ideally we would not always send these capabilities, they would be up to the developer using FoxgloveServer to enable or disable them. If someone wants to use this class but doesn't want to offer parameter support, they should be able to suppress the capability from being sent.

Reply:
I agree. I will refactor it once this PR is in